### ISSUE TITLE
Mark a snapshot to not be exclude

### ISSUE THREAD

**Issue by DanielVenturini** [teejee2008/timeshift#572]
Hello, nice app.

It'd be great if one snapshot could be marked not to be excluded. I'd like to have the first snapshot always available.

**Comment by camilova** [comment]
Did you mind that we could flag a snapshot to prevent this to be auto removed/destroyed?
If so, I like this idea too!

**Comment by DanielVenturini** [comment]
That's right. The snapshots are removed automatically by timeshift, because they spend a lot of memory size. But, I'd like to select some snapshots to be always available.
